destination/format/csv.go

```diff
@@ -489,3 +478,24 @@ func isDateOrTimeType(in string) bool {
 		return false
 	}
 }
+
+func mapAvroToSnowflake(field *avro.Field) string {
+	p := field.Type().(*avro.PrimitiveSchema)
+	switch p.Logical().Type() {
```

Comment:
Can Logical() return nil?

Reply:
yeah, I just got a nil pointer here 🤣 gonna fix this, thanks